sanic_ext/extensions/logging/extractor.py

```diff
@@ -0,0 +1,116 @@
+import logging
+
+from typing import Any, Dict, Optional, TypedDict
+
+
+class LoggerConfig(TypedDict):
+    level: str
+    propagate: bool
+    handlers: list[str]
```

Comment:
```suggestion
from typing import Any, Dict, List, Optional, TypedDict


class LoggerConfig(TypedDict):
    level: str
    propagate: bool
    handlers: List[str]
```

Simple fix for the CI failure